cmd/kata-pkgsync/cli/main.go

```diff
@@ -0,0 +1,231 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/marcov/obsgo"
+	"github.com/sirupsen/logrus"
+	pb "gopkg.in/cheggaaa/pb.v1"
+)
+
+const (
+	MYNAME         = "kata-pkgsync"
+	defaultConfig  = "config.yaml"
+	defaultOBSDest = "obs-packages"
+)
+
+func usage() {
+	fmt.Fprintf(flag.CommandLine.Output(), `NAME:
+   %s - Synchronize packages from OBS to Packagecloud
+
+USAGE:
+   %s [options] [config file]
+
+By default %s reads the configuration from a file named %s. This can be
+overridden by passing the path to a config file.
+
+Options:
+`, MYNAME, MYNAME, MYNAME, defaultConfig)
+
+	flag.PrintDefaults()
+}
+
+func main() {
+	flag.Usage = usage
+	verbose := flag.Bool("debug", false, "debug mode")
+	dlPath := flag.String("dir", defaultOBSDest, "Destination directory of packages download from OBS")
+	dryRun := flag.Bool("dry-run", false, "dry-run mode (do not download/upload files)")
```

Comment:
That's a good idea, we just need to figure out a way of passing OBS and Packagecloud crendentials to the CI.

Reply:
Jenkins has a secure store for creds, which it can then inject or make accessible into the job environments - you might want to see if somebody has figured out what sort of cred to store in jenkins already.